Some missing includes wiht musl for open, write and inet_addr.
--- a/src/chm_http.c
+++ b/src/chm_http.c
@@ -43,12 +43,15 @@
 #include <sys/socket.h>
 #include <sys/types.h>
 #include <netinet/in.h>
+#include <arpa/inet.h>
 
 /* threading includes */
 #include <pthread.h>
 
 #include <getopt.h>
 
+#include <unistd.h>
+
 int config_port = 8080;
 char config_bind[65536] = "0.0.0.0";
 

